File: classes/parte-5-funciones-y-modularidad/077-multiples-retornos-y-desestructuracion/implementaciones/php/main.php
<?php
function divmod($a, $b) {
    return [intdiv($a, $b), $a % $b];
}

[$a, $b] = preg_split('/\s+/', trim(fgets(STDIN)));
[$q, $r] = divmod((int) $a, (int) $b);
echo "cociente=$q resto=$r\n";

File: classes/parte-5-funciones-y-modularidad/077-multiples-retornos-y-desestructuracion/implementaciones/go/main.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strconv"
	"strings"
)

func divmod(a, b int) (int, int) {
	return a / b, a % b
}

func main() {
	line, _ := bufio.NewReader(os.Stdin).ReadString('\n')
	f := strings.Fields(line)
	a, _ := strconv.Atoi(f[0])
	b, _ := strconv.Atoi(f[1])
	q, r := divmod(a, b)
	fmt.Printf("cociente=%d resto=%d\n", q, r)
}
